source/overview/faq.rst

```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following chart highlights the end user features of Mattermost and their support across Web, Desktop and Mobile applications (iOS and Android).
+
+.. csv-table::
+    :header: "Feature", "Web", "Desktop", "Mobile"
+
+    "**Channels**"
+    "Create a new channel", "X", "X", "X"
+    "Join a channel", "X", "X", "X"
+    "Leave a channel", "X", "X", "X"
+    "Favorite a channel", "X", "X", "X"
+    "Mute a channel", "X", "X", "X"
+    "Manage members", "X", "X", "X"
+    "Add members", "X", "X", "X"
+    "Edit channel", "X", "X", "X"
+    "Archive members", "X", "X", "X"
+    **Messages**
+    "Threaded messages", "X", "X", "X"
+    "Markdown", "X", "X", "O"
+    "Emojis", "X", "X", "X"
+    "Emoji reactions", "X", "X", "X"
+    "Viewing emoji reactions", "X", "X", "v1.13"
+    "File sharing", "X", "X", "X"
+    "@ mentions", "X", "X", "X"
+    "Hashtags", "X", "X", "v1.13"
+    "Search (with in:, from:, before:, on: and after:)", "X", "X", "X"
+    "Search highlighting", "X", "X", "v1.15"
+    "View/marking pinned or flagged posts", "X", "X", "X"
+    "Image link previews", "X", "X", "X"
+    "Website previews", "X", "X", "X"
+    "Notifications", "X", "X", "X"
+    **Integrations**
+    "Slash commands", "X", "X", "S"
+    "Server-side plugins", "X", "X", "X"
+    "User interface plugins", "X", "X",
+    "Oauth applications", "X", "X", "X"
+    "Incoming webhooks", "X", "X", "X"
+    "Outgoing webhooks", "X", "X", "X"
+    "Message attachments", "X", "X", "X"
+    "Message buttons", "X", "X", "X"
+    "Message menus", "X", "X", "X"
+    **Teams**
+    "Multi-team support for notifications", "X", "X", "X"
+    "Team switching", "X", "X", "X" 
+    "Team-based theming", "X", "X", "X"
+    "Create a new team", "X", "X",
+    "Join existing team", "X", "X", "X"
+    "Get team invite link", "X", "X",
+    "Add members to team", "X", "X",
+    "Manage members", "X", "X",
+    "Leave team", "X", "X",
+    **Other**
+    "Custom user interface themes", "X", "X", "O"
+    "Localization for 15 languages", "X", "X", "X"
+    "Account settings", "X", "X", "O"
+    "OAuth login with Google/Office 365", "X", "X",
+    "Connect to multiple Mattermost servers from a single interface and switch with shortcut keys (similar to tabs in web browser", " ", "X", 
+    "Auto-start Mattermost when a user logs into their machine", " ", "X"    
+    "(Windows) Add Mattermost to Start menu, taskbar and system tray", " ", "X", 
+    "(Windows/Mac) Deep link to the desktop app via mattermost:// protocol if app is already installed", " ", "X", 
+    "(Mac) Add Mattermost to the applications Dock", " ", "X", 
+    "(Linux) Desktop Entry for the application to more easily integrate into a desktop environment", " ", "X", 
+
+**Notes:**
+
+* X - Full Support
+
+* S - Support for ‘collapse’, ‘expand’, ‘join’, ‘open’, ‘leave’, ‘logout’, ‘msg’, ‘grpmsg’, ‘rename’, ‘invite_people’, ‘remove’ not yet included
```

Comment:
I'm curious - is "S" in this case equivalent to "O" -- given O implies partial support.